### PR TITLE
Add parameter IP Address to VPE

### PR DESCRIPTION
**Pull request by VaishnaviGopal**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/resource_ibm_is_virtual_endpoint_gateway.go [hunk 1]
```diff
@@ -118,13 +118,17 @@ func resourceIBMISEndpointGateway() *schema.Resource {
 						isVirtualEndpointGatewayIPsSubnet: {
 							Type:        schema.TypeString,
 							Optional:    true,
-							Computed:    true,
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway.go:121]: why is this removed?

Comment (VaishnaviGopal) [reply ibm/resource_ibm_is_virtual_endpoint_gateway.go:121]: Because.. there the get method is not having a parameter for `subnet` to set. 
https://github.com/VaishnaviGopal/terraform-provider-ibm/blob/14f456aceb64744c7e3be9fb0745704a87e08e0d/ibm/resource_ibm_is_virtual_endpoint_gateway.go#L401
